### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by jrgarou**
### Header 
It's still a mockup
https://github.com/Paratii-Video/paratii-portal/blob/5349dc4c0b0f3c007c12a626389fa3d1b61493a9/src/scripts/components/pages/Landing.js#L203-L219

image

### Videos list
Using the `getUploaderVideos` from `selectors/UploaderSelectors` to show some videos(or not)
https://github.com/Paratii-Video/paratii-portal/blob/5349dc4c0b0f3c007c12a626389fa3d1b61493a9/src/scripts/components/pages/Landing.js#L289-L302
image

### CODE REVIEW

File: src/scripts/components/widgets/FilesUploader.js [hunk 1]
```diff
@@ -131,58 +123,58 @@ class FilesUploader extends Component<Props, Object> {
...
-        dragClass: klass
-      })
+    const status = e.type
+    let klass = ''
+    if (status === 'dragenter' || status === 'mouseover') {
+      klass = 'dragenter'
+    } else if (status === 'drop') {
+      klass = 'drop'
     } else {
-      this.props.checkUserWallet()
```
Comment (eliawk) [inline src/scripts/components/widgets/FilesUploader.js]: why do remove this control? without that the user can upload even if is not logged in

**Comment by eliawk** [comment]
the card layout has changed
schermata 2018-06-06 alle 09 30 51

File: src/scripts/components/structures/Main.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import styled from 'styled-components'
 
 type Props = {
+  landing: Boolean,
```
Comment (eliawk) [inline src/scripts/components/structures/Main.js]: `Boolean` uppercase is an Object, but we just need a `boolean`

File: src/scripts/components/widgets/FilesUploader.js [hunk 3]
```diff
@@ -131,58 +123,58 @@ class FilesUploader extends Component<Props, Object> {
...
-          onClick={this.onCheck}
-          onChange={this.onFileChosen}
-          onDragEnd={this.onDrag}
-        />
-        <UploadCover>
-          {card && <Title>Upload video</Title>}
-          <UploadCoverIcon>
-            {card && (
-              <Icon>
-                <FilesUploaderSvg />
```
Comment (eliawk) [inline src/scripts/components/widgets/FilesUploader.js]: removing this the card layout is now missing the svg icon
schermata 2018-06-06 alle 09 30 51

File: src/scripts/components/widgets/FilesUploader.js (hunk 1 above)
Comment (jrgarou) [reply src/scripts/components/widgets/FilesUploader.js]: hum. I don't know why. Let me fix

File: src/scripts/components/widgets/FilesUploader.js (hunk 3 above)
Comment (jrgarou) [reply src/scripts/components/widgets/FilesUploader.js]: My bad, @eliawk 
Fixing that

**Pull request merged**